torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -78,25 +78,26 @@ RegisterOperators reg({
           };
         }),
     Operator(
-        "prim::TensorToNum(Tensor a) -> Scalar",
+        "prim::Int(Tensor a) -> int",
+        [](const Node* node) -> Operation {
+          return [](Stack& stack) {
+            at::Tensor a;
+            pop(stack, a);
+            at::OptionalDeviceGuard guard(device_of(a));
```

Comment:
Why is this device guard needed ? There isn't a new tensor being created

Reply:
Looks like old code...